pac4j-saml/src/main/java/org/pac4j/saml/sso/impl/SAML2WebSSOMessageSender.java

```diff
@@ -119,7 +123,7 @@ protected final void invokeOutboundMessageHandlers(final SPSSODescriptor spDescr
                 final SAMLOutboundProtocolMessageSigningHandler handler = new
                         SAMLOutboundProtocolMessageSigningHandler();
                 handler.setSignErrorResponses(this.signErrorResponses);
-                handler.invoke(outboundContext);
+                //handler.invoke(outboundContext);
```

Comment:
I'm surprised to see that line commented. I had in mind that this was were the work was done...